packages/fxa-settings/src/pages/InlineTotpSetup/index.tsx

```diff
@@ -0,0 +1,320 @@
+                    href="https://support.mozilla.org/kb/secure-firefox-account-two-step-authentication"
+                  >
+                    these authentication apps
+                  </LinkExternal>
+                ),
+              }}
+            >
+              <p className="text-sm">
+                Add a layer of security to your account by requiring security
+                codes from one of{' '}
+                <LinkExternal
+                  className="link-blue text-sm"
+                  href="https://support.mozilla.org/kb/secure-firefox-account-two-step-authentication"
+                >
+                  these authentication apps
+                </LinkExternal>
+                .
+              </p>
+            </FtlMsg>
+            <button
+              type="submit"
+              className="cta-primary cta-xl w-full m-2 my-6"
+              onClick={() => {
+                // TODO: Add in any further functionality here
+                setShowIntro(false);
+              }}
+            >
+              <FtlMsg id="inline-totp-setup-continue-button">Continue</FtlMsg>
+            </button>
+            <button type="button" className="link-blue text-sm">
+              <FtlMsg id="inline-totp-setup-cancel-setup-button">
+                Cancel setup
+              </FtlMsg>
+            </button>
+          </section>
+        </>
+      )}
+      {!showIntro && (
+        <>
+          {showQR ? (
+            <CardHeader
+              headingText="Scan authentication code"
+              headingWithCustomServiceFtlId="inline-totp-setup-show-qr-custom-service-header"
+              headingWithDefaultServiceFtlId="inline-totp-setup-show-qr-default-service-header"
+              {...{ serviceName }}
+            />
+          ) : (
+            <CardHeader
+              headingText="Enter code manually"
+              headingWithCustomServiceFtlId="inline-totp-setup-no-qr-custom-service-header"
+              headingWithDefaultServiceFtlId="inline-totp-setup-no-qr-default-service-header"
+              {...{ serviceName }}
+            />
+          )}
+          <section>
+            <form noValidate>
+              <div id="totp" className="totp-details">
+                {showQR ? (
+                  <>
+                    <FtlMsg
+                      id="inline-totp-setup-use-qr-or-enter-key-instructions"
+                      elems={{
+                        toggleToManualModeButton: (
+                          <button
+                            type="button"
+                            onClick={() => {
+                              setShowQR(false);
+                            }}
+                            className="link-blue inline"
+                          >
+                            Can’t scan code?
+                          </button>
+                        ),
+                      }}
+                    >
+                      <p className="text-sm m-4">
```

Comment:
I'm not a big fan of margins all the way around since the left/right hand space then differs per page hmm, can we change all of these to `mb-4` instead?

<img width="469" alt="image" src="https://user-images.githubusercontent.com/13018240/216117117-4fef2ae4-0bab-42b4-b3bd-c97117bb7e68.png">